app/models/purchase.py

```diff
@@ -45,23 +45,54 @@ class Order(db.Model):
     date = db.Column(db.DateTime)
     status = db.Column(db.Integer)
 
-    def __init__(self, cart_items):
-        self.date = datetime.now(pytz.timezone('US/Eastern'))
+    vendor_id = db.Column(db.Integer)
+    company_name = db.Column(db.String(64))
+
+    def __init__(self, date, vendor_id):
         self.status = Status.PENDING
+        self.date = date
```

Comment:
why are we passing in date instead of using the time when the order is created in the db?